ods_ci/tests/Resources/CLI/DataSciencePipelines/DataSciencePipelinesBackend.resource

```diff
@@ -63,10 +63,6 @@ Create PipelineServer Using Custom DSPA
     ...    ${assert_install}=${TRUE}    ${configure_pip_index}=${TRUE}
```

Comment:
maybe this argument is not needed anymore now? could you remove it in case?

Reply:
https://github.com/red-hat-data-services/ods-ci/issues/1959